bundle/bundle.go

```diff
@@ -343,6 +413,38 @@ func (r *Reader) Read() (Bundle, error) {
 	return bundle, nil
 }
 
+func (r *Reader) checkSignaturesAndDescriptors(signatures *SignaturesConfig, descriptors []*Descriptor) error {
+	if r.skipVerify {
+		return nil
+	}
+
+	if signatures == nil && r.verificationConfig != nil {
+		return fmt.Errorf("bundle reader provided with signature verification config but .signatures.json file not included in the bundle")
```

Comment:
Similar to the other comments I've left, I think we can simplify some of these errors messages. E.g., here we could drop the term "reader" (it's an implementation detail) and just combine explicitly state what's wrong:

```bundle missing .signatures.json file```

Reply:
Updated